oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -106,20 +106,25 @@ file=sys.stderr)
                     print("Pulling {} from the primary node.".format(e))
                     url = "https://{}:{}/cmapi/{}/node/meta/{}".format(primary_address, \
 api_port, api_version, e)
-                    r = requests.get(url, verify=False, headers=headers, timeout=2)
+                    r = requests.get(url, verify=False, headers=headers, timeout=30)
```

Comment:
Let's be safe and increase this. Saw some timeouts sporadically when retrieving metadata